cluster/gce/config-default.sh

```diff
@@ -167,3 +167,6 @@ EVICTION_HARD="${EVICTION_HARD:-memory.available<100Mi,nodefs.available<10%,node
 
 # Optional: custom scheduling algorithm
 SCHEDULING_ALGORITHM_PROVIDER="${SCHEDULING_ALGORITHM_PROVIDER:-}"
+
+# Optional: Enable DNS horizontal autoscaler
+ENABLE_DNS_HORIZONTAL_AUTOSCALER="${KUBE_ENABLE_DNS_HORIZONTAL_AUTOSCALER:-true}"
```

Comment:
I was intended to only make it a default on gce, but not sure whether I should do this. @thockin